pkg/cmd/importcmd/import.go

```diff
@@ -1446,6 +1446,9 @@ func (options *ImportOptions) GetGitRepositoryDetails() (*gits.CreateRepoData, e
 	if err != nil {
 		return nil, err
 	}
+	//config git repositoryoptions parameters: Owner and RepoName
+	options.GitRepositoryOptions.Owner = options.Organisation
```

Comment:
I am wondering whether this not has to happen much earlier? Basically in `Run`. If you look where and how `GetGitRepositoryDetails` is called, I wonder what happens when you specify `--batch` as well:

```go
	if !o.BatchMode {
		details, err = o.GetGitRepositoryDetails()
		if err != nil {
			return err
		}

		o.Filter.ProjectName = details.RepoName
	}
```

Reply:
thanks for your information, I doubled checked BatchMode and this change is not necessary for Batchmode because:
1. Batchmode use " --project-name" options ("o.Filter.ProjectName") as project/repo Name, only when batchmode equals false, the option "name" is used. you can also see this from: 

		o.Filter.ProjectName = details.RepoName
2. Batchmode also use "--org" options, but this already config at "ImportCreatedProject" in import.go, but organization popup happened before this config, so the issue only impact when batchmode equals false.
         Row #384 in Run() and Row #614 in func CreateNewRemoteRepository()

		o.Filter.ProjectName = details.RepoName
          
		options.GitRepositoryOptions.Owner = options.GetOrganisation()
3. Changing in function GetGitRepositoryDetails() also because there are total 3 files call this function, so this will fix same issue for create mlquickstart and create spring.